migration/migrator/data/submitty_db.sql

```diff
@@ -658,6 +658,21 @@ CREATE TABLE public.terms (
 );
 
 
+--
+-- Name: unverified_users; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.unverified_users (
+    user_id character varying(50) NOT NULL,
+    user_givenname character varying NOT NULL,
+    user_password character varying,
```

Comment:
is this plaintext password or a salted/hashed password?
(maybe we should change the variable name, but let's not if this is already the standard name we use in other tables)